spark_pipeline_framework/transformers/fhir_receiver/v1/fhir_receiver.py

```diff
@@ -466,6 +476,21 @@ def _transform(self, df: DataFrame) -> DataFrame:
         with ProgressLogMetric(
             name=f"{name}_fhir_receiver", progress_logger=progress_logger
         ):
+            # if additional_parameters_view is given,
+            # this is for generic use. e.g. to support blocking by adding search parameters in a query to FHIR
+            if additional_parameters_view and additional_parameters:
```

Comment:
should we support either additional_paramaters or additional_parameters_view being passed in and not just AND

Reply:
Then your code (I think) requires the caller to pass in both